packages/accounts-password/password_tests.js

```diff
@@ -1789,6 +1789,26 @@ if (Meteor.isServer) (() => {
       ]);
     });
 
+    Tinytest.addAsync("passwords, replace email", async test => {
+      const origEmail = `originalemail@test.com`;
+      const userId = await Accounts.createUser({
```

Comment:
Should it be `createUserAsync` here? Also, does this test actually work? It seems createUser only provides the ID via a callback, not directly.